royal/ext/sqla.py

```diff
@@ -0,0 +1,168 @@
+import logging
+
+from pyramid.location import lineage
+import royal
+from sqlalchemy.orm.collections import MappedCollection
+
+log = logging.getLogger(__name__)
+
+
+def includeme(config):
+    config.scan(__name__)
+
+
+class Collection(royal.Collection):
+    sa_model = None
+    sa_exceptions = None
+    entity_cls = None
```

Comment:
Ok I get it: dependency injection? Should we define a clean interface of `sa_model`?